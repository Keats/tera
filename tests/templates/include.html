<html>
  <head>
    <title>Products</title>
  </head>
  <body>
    <div>
      {% include "included.html" %}
    </div>
  </body>
</html>
