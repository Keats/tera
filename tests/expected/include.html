<html>
  <head>
    <title>Products</title>
  </head>
  <body>
    <div>
      <table>
        <thead>
          <th>User</th><th>Number of reviews</th>
        </thead>
        <tr>
          <td>bob</td><td>2</td>
        </tr>
      </table>
    </div>
  </body>
</html>
